khelo/lib/ui/flow/matches/match_detail/components/match_detail_squad_view.dart

```diff
@@ -53,16 +53,18 @@ class MatchDetailSquadView extends ConsumerWidget {
         [];
 
     final firstTeamBench = firstTeam?.team.players
-            ?.where((element) =>
+            .where((element) =>
                 !firstTeamSquad.map((e) => e.id).contains(element.id) &&
-                element.isActive)
+                element.user!.isActive)
+            .map((e) => e.user!)
             .toList() ??
         [];
 
     final secondTeamBench = secondTeam?.team.players
-            ?.where((element) =>
+            .where((element) =>
                 !secondTeamSquad.map((e) => e.id).contains(element.id) &&
-                element.isActive)
+                element.user!.isActive)
```

Comment:
on what basis we're force unwrapping user? if we would manage user this way, we have to force unwrap or check for null  at every places which is not right thing to do as per my opinion